src/types/models/dealer.ts

```diff
@@ -0,0 +1,18 @@
+export interface DealerLocation {
+  address: string
+  city: string
+  zipCode: string
+}
+
+export interface DealerCarouselPromotion {
```

Comment:
Not sure but as it's the forntend part of the dealer Promition perhaps we can unify the type with the ones we have for dealerhub
https://github.com/carforyou/carforyou-api-client-pkg/blob/master/src/types/models/dealerPromotion.ts

Reply:
is a but unclear tbh. i tried to follow the pattern we used for listingsSearch and this is what  came up with. This is what happening when we work on the same feature, on the same apis, but not talking...will see what i can do

Reply:
As we consume the promotion in frontend I thought we might be able to use the same type as we have in dealerhubg but you will need to check that, on the first glance it looked like to logo is missing atm probably as it's not implemented in dealerhub...

Reply:
no, as far as i know is not implemented in DH. There is this ticket https://autoricardo.atlassian.net/browse/CAR-4710
Makes total sense to have the same type. TBH i was not aware of the existing ones. Good that you pointed that out.

Reply:
i can put them in the same file, but more than that i don't think i can do more. wdyt?

Reply:
Yeah think thats a good start then we will spot more easy if they become unified 😄

Reply:
will do like that then 👍

Reply:
done